lib/commands/generate-skeletons/command.js

```diff
@@ -1,9 +1,86 @@
-'use strict';
 const UI = require('../../ui').UI;
 const fs = require('../../file-system');
+const _ = require('lodash');
```

Comment:
did we already have this dependency? what are we using it for?

Reply:
I used lodash in various places to avoid long tedious array modification.

It's not in explicit dependencies, but it was in some deep dep tree of some other deps we have.